about.html

```diff
@@ -0,0 +1,120 @@
+            <ul class="second-nav-content ">
+                <li><a href="" class="links"></a>About</li>
+                <li><a href="" class="links"></a>Program </li>
+                <li><a href="" class="links"></a>Join</li>
+                <li><a href="" class="links"></a>Sponsor</li>
+                <li><a href="" class="links"></a>News</li>
+                <button class="delete"  onclick="closeMobileMenu()">
+                </button>
+                <button class="nav-button"><a href=""></a>FIFA compaign</button>
+            </ul>
+        </nav>
+        <button onclick="openMobileMenu()" class="toolbar" >
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon-menu" width="44" height="44" viewBox="0 0 24 24"
+              stroke-width="1.5" stroke="#272a32" fill="none" stroke-linecap="round" stroke-linejoin="round">
+              <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+              <line x1="4" y1="6" x2="20" y2="6" />
+              <line x1="4" y1="12" x2="20" y2="12" />
+              <line x1="4" y1="18" x2="20" y2="18" />
+            </svg>
+          </button>
+    </header>
+    <section class="about about-1">
+        <div>
+            <h1>WELCOME TO FIFA WORLD <br> CUP QATAR 2022 </h1>
+        </div>
+        <div class="headlin-description about-description">
+            <p>
+                The new FIFA is modernising football to be global, accessible and inclusive in all aspects. Not just on one or two continents, but everywhere.
+
+                Under our vision to make football truly global, we will help develop football everywhere so that there are at least 50 national teams and 50 clubs from all continents that can compete at a top competitive level.            </p>
+        </div>
+        <div class="contact">
+            <p>
+                Please contact us via Email for any further questions about FIFA &
+                QATAR world cup 2022!
+              </p>
+              <a href="#">worldcup@FIFA.com </a>
+        </div>
+    </section>
+   <section class="main-about">
+        <h2>ALL FIFA</h2>
+        <p>
+            FIFA is working on a wide range of topics from improving the women's game and developing football around the world to ensuring better football governance structures.
+        </p>
+        <div class="logo-about">
+            <img src="./images/six.webp" alt="">
+        </div>
+        <h2>FIFA Council</h2>
+        <p>
+            The Bureau of the FIFA Council decided to lift the suspension that was imposed on the Football Kenya Federation (FKF) in February 2022 in accordance with article 16 of the FIFA Statutes.
+        </p>
+
+        <div class="details-about-2">
+            <div class="right">
+                <img src="./images/seven.webp" alt="">
+            </div>
+            <div class="left">
+                <img src="./images/eight.webp" alt="">
+            </div>
+        </div>
+
+   </section>
```

Comment:
- [ ] If you look at the sample template for the `About` Page, there are background images used in this section. Your design has no background image. Kindly use the background image for both the desktop and mobile versions of your project. 👍 

### Your Design - Desktop

![image](https://user-images.githubusercontent.com/77106962/205458035-04a1976f-4c4a-4515-9155-e0a0a36bf46a.png)

### Sample Template - Desktop

![image](https://user-images.githubusercontent.com/77106962/205458072-82a09109-98a9-41e8-ba1d-fd2e2483f6af.png)

### Your Design - Mobile

![image](https://user-images.githubusercontent.com/77106962/205458093-28f8d1ef-828a-46c5-a922-c9c9d510fe23.png)


### Sample Template - Mobile

![image](https://user-images.githubusercontent.com/77106962/205458121-bdf87f89-1502-4d6a-a50e-043bd17c455d.png)